packages/fetchai/connections/p2p_libp2p/connection.py

```diff
@@ -150,7 +150,7 @@ async def start(self):
         self._thread.start()
         try:
             return await asyncio.shield(self.future)
-        except asyncio.CancelledError:
+        except asyncio.CancelledError:  # pragma: no cover
```

Comment:
on my local machine, this is already covered by  `tests/test_packages/test_connections/test_p2p_libp2p/test_errors.py::TestP2PLibp2pConnectionFailureGolangBuild::test_timeout` 
(https://github.com/fetchai/agents-aea/blob/develop/tests/test_packages/test_connections/test_p2p_libp2p/test_errors.py#L67)